examples/demos/create_infrastructure_with_storage.pp

```diff
@@ -0,0 +1,288 @@
+################################################################################
+# (C) Copyright 2020 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+# This example works for api_variant "Synergy"
+# Create FC Network
+oneview_fc_network{'Test Puppet FC Network':
+  ensure => 'present',
+  data   => {
+    name                    => 'Test Puppet FC Network',
+    connectionTemplateUri   => nil,
+    autoLoginRedistribution => true,
+    fabricType              => 'FabricAttach'
+    }
+}
+
+# Create Ethernet Network
+oneview_ethernet_network{'Test Puppet Network':
+  ensure => 'present',
+  data   => {
+    name           => 'Test Puppet Network',
+    vlanId         => '1045',
+    purpose        => 'General',
+    smartLink      => true,
+    privateNetwork => false
+  }
+}
+
+oneview_storage_system{'storage_system_1':
```

Comment:
Follow a uniform naming convention. Since you are using Test * *. Please follow the same throughout.

Reply:
You can use any naming convention. Just make it uniform. Don't mix underscore lower case uppercase, camel, together

Reply:
done